spec/frontman/context_spec.rb

```diff
@@ -25,19 +25,29 @@
     end
 
     it 'should return not react to unknown content' do
-      expect(subject.content_for?('unknown content')).to eq false
+      key = 'unknown content'
+
+      expect(subject.content_for?(key)).to eq false
+
+      expect { subject.append_content(key) }.to raise_error NoMethodError
     end
 
-    it 'should return content when it\'s set' do
+    it 'should react to known content' do
       Frontman::App.instance.current_page = resource
-      subject.content_for('my-key', 'myvalue')
-      expect(subject.yield_content('my-key')).to eq 'myvalue'
+      key = 'my-key'
+      subject.content_for(key, 'myvalue')
+      expect(subject.content_for?(key)).to eq true
     end
 
-    it 'should react to known content' do
+    it 'should return content when it\'s set' do
       Frontman::App.instance.current_page = resource
-      subject.content_for('my-key', 'myvalue')
-      expect(subject.content_for?('my-key')).to eq true
+      key = 'my-key'
+
+      subject.content_for(key, 'myvalue')
+      expect(subject.yield_content(key)).to eq 'myvalue'
+
+      subject.append_content(key, ',othervalue')
```

Comment:
Could you please add a test where we use `append_content` with a block as well? This will test the scenario where we use this method in, for example, templates.

Reply:
I have added the block tests however I am unable to test the outcome because of this code which requires erb/haml template. Does this work in other templating languages?

https://github.com/westonganger/frontman/blob/7f57e7d6c6bd39a7bc8ba70ad5fe3162fa94ef7d/lib/frontman/context.rb#L149-L152

Reply:
Ah yes, you're right; this behavior is specific to templates. I'm not sure it'll work for other templating languages, so maybe we can test this by creating a template in the `mocks` directory in that case, what do you think?

Reply:
Ok. Done.